types/react-native-extra-dimensions-android/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-native-extra-dimensions-android 1.2
+// Project: https://github.com/Sunhat/react-native-extra-dimensions-android
+// Definitions by: Kiryl Arlouski <https://github.com/kirusick>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```

Comment:
```suggestion
// TypeScript Version: 3.0
```
why do you need 3.6? The RN depends on 3.0 AFAIK

Reply:
I didn't know this fact. Fixed